sample-app/redisearch-node-rest/server.js

```diff
@@ -27,72 +24,32 @@ app.get('/api/1.0/movies/search', (req, res) => {
 
   if (sortBy) {
     options.sortBy = sortBy;
-    options.ascending = true; // if sorted by default it is ascending
+    options.ascending = true; // Default to sort ascending.
   }
 
   if (ascending) {
-    options.ascending = (ascending==1 || ascending.toLocaleLowerCase()==='true');
+    options.ascending = (ascending ==1 || ascending.toLocaleLowerCase() === 'true');
   }
   
-  searchService.search(
-    queryString,            // query string
-    options,                // options
-    function(err, result){  // callback
-      res.json(result);
-    }
-  );
-})
-
-app.get('/api/1.0/movies/group_by/:field', (req, res) =>{
-  searchService.getMovieGroupBy(req.params.field, function(err, result){
-    res.json(result);
-  });
+  return res.json(await searchService.search(queryString, options));
 });
 
-app.get('/api/1.0/movies/:id', (req, res) =>{
-  searchService.getMovie(req.params.id, function(err, result){
-    res.json(result);
-  });
-});
+app.get('/api/1.0/movies/group_by/:field', async (req, res) => res.json(await searchService.getMovieGroupBy(req.params.field)));
```

Comment:
https://www.npmjs.com/package/express-promise-router can use that to clean the code a bit